Author: Daniel Kahn Gillmor <dkg@fifthhorseman.net>
Subject: Drop Windows builds

On debian packages for Sequoia crates, we do not package the Windows
variants.  debian packages are for building tooling for debian systems.

--- sequoia-net/Cargo.toml
+++ sequoia-net/Cargo.toml
@@ -85,7 +85,6 @@
 compression = ["sequoia-openpgp/compression"]
 compression-bzip2 = ["sequoia-openpgp/compression-bzip2"]
 compression-deflate = ["sequoia-openpgp/compression-deflate"]
-crypto-cng = ["sequoia-openpgp/crypto-cng"]
 crypto-nettle = ["sequoia-openpgp/crypto-nettle"]
 default = ["sequoia-openpgp/default"]
 [badges.gitlab]
